modules/deploy/pages/deployment-option/self-hosted/kubernetes/aks-guide.adoc

```diff
@@ -43,18 +44,78 @@ NOTE: The Helm chart configures `podAntiAffinity` rules to make sure that only o
 az group create --name redpandaResourceGroup --location eastus
 ----
 
-. Create an AKS cluster with three worker nodes (one for each Redpanda broker):
+. Create an AKS cluster:
 +
 [,bash]
 ----
-az aks create -g redpandaResourceGroup -n redpanda \
+az aks create -g redpandaResourceGroup -n <cluster-name> \
```

Comment:
If we're not going to be prescriptive with the name, can we please use environment variables throughout? It makes the instructions copy/pasteable.
```suggestion
az aks create -g redpandaResourceGroup -n $CLUSTER_NAME \
```

Reply:
@joejulian our placeholders are editable so users can edit before copying